Date:   Fri Jan 4 04:07:29 2019 -0800

    Add a couple more symlinks and stuff for block-device mounting
---
 .../vmware-workstation/vmware-workstation-15.0.2.10952284-r2.ebuild    | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/app-emulation/vmware-workstation/vmware-workstation-15.0.2.10952284-r2.ebuild b/app-emulation/vmware-workstation/vmware-workstation-15.0.2.10952284-r2.ebuild
index 89c24b1..5e83794 100644
--- a/app-emulation/vmware-workstation/vmware-workstation-15.0.2.10952284-r2.ebuild
+++ b/app-emulation/vmware-workstation/vmware-workstation-15.0.2.10952284-r2.ebuild
@@ -482,12 +482,13 @@ src_install() {
 
 	# create symlinks for the various tools
 	local tool ; for tool in thnuclnt vmware vmplayer{,-daemon} licenseTool vmamqpd \
-			vmware-{app-control,enter-serial,gksu,fuseUI,hostd,modconfig{,-console},netcfg,tray,unity-helper,vim-cmd,vmblock-fuse,vprobe,wssc-adminTool,zenity} ; do
+			vmware-{app-control,enter-serial,gksu,fuseUI,hostd,modconfig{,-console},netcfg,tray,unity-helper,vim-cmd,vmblock-fuse,vmware-fuseUI,vprobe,wssc-adminTool,zenity} ; do
 		dosym appLoader "${VM_INSTALL_DIR}"/lib/vmware/bin/"${tool}"
 	done
 	dosym "${VM_INSTALL_DIR}"/lib/vmware/bin/vmplayer "${VM_INSTALL_DIR}"/bin/vmplayer
 	dosym "${VM_INSTALL_DIR}"/lib/vmware/bin/vmware "${VM_INSTALL_DIR}"/bin/vmware
 	dosym "${VM_INSTALL_DIR}"/lib/vmware/bin/vmware-netcfg "${VM_INSTALL_DIR}"/bin/vmware-netcfg
+	dosym "${VM_INSTALL_DIR}"/lib/vmware/bin/vmware-fuseUI "${VM_INSTALL_DIR}"/bin/vmware-fuseUI
 	dosym "${VM_INSTALL_DIR}"/lib/vmware/icu /etc/vmware/icu
 	dosym "${VM_INSTALL_DIR}"/lib/vmware-installer/${vmware_installer_version}/vmware-installer "${VM_INSTALL_DIR}"/bin/vmware-installer
 

